src/box/box.h

```diff
@@ -599,6 +599,12 @@ box_sequence_reset(uint32_t seq_id);
 API_EXPORT int
 box_session_push(const char *data, const char *data_end);
 
+/**
+ * \return current session's unique monotonic identifier (\sa box.session.id)
+ */
+API_EXPORT uint64_t
+box_session_id();
```

Comment:
Nit: Should be defined/declared as `box_session_id(void)` (otherwise it can take an arbitrary number of arguments in C).

Reply:
Fixed: added `void` type parameter.